### ISSUE TITLE
hyperparameters

### ISSUE THREAD

**Issue by davidenitti** [XinJingHao/PPO-Discrete-Pytorch#1]
the results you have obtained in the plots are with the default parameters in the main? if not, which parameters do you use?
thanks

**Comment by XinJingHao** [comment]
Yes, the default parameters in main.py

**Comment by davidenitti** [comment]
thanks, by any chance do you know the difference with PPO of mushroomrl? trying to reproduce with that library but I can't. I'm trying to match everything, the reward clip you do, the parameters, and so on. Your code seems to converge much faster.

**Comment by XinJingHao** [comment]
Sorry, I have no idea about mushroomrl.

----- 原始邮件 -----
发件人: "Davide Nitti" ***@***.***>
收件人: "XinJingHao/PPO-Discrete-Pytorch" ***@***.***>
抄送: "XinJingHao" ***@***.***>, "Comment" ***@***.***>
发送时间: 星期一, 2023年 5 月 08日 下午 11:48:28
主题: Re: [XinJingHao/PPO-Discrete-Pytorch] hyperparameters (Issue #1)

thanks, by any chance do you know the difference with PPO of mushroomrl? trying to reproduce with that library but I can't. I'm trying to match everything, the reward clip you do, the parameters, and so on. Your code seems to converge much faster.

-- 
Reply to this email directly or view it on GitHub:
https://github.com/XinJingHao/PPO-Discrete-Pytorch/issues/1#issuecomment-1538622255
You are receiving this because you commented.

Message ID: ***@***.***>
-- 
祝好！

-------------------------
上海交通大学电子信息与电气工程学院
辛靖豪
